react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,26 @@
-import { isArray } from './utils';
+import isArray from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+const returnOneArray = (array) => array.reduce((acc, val) => acc.concat(val), []);
+
+function min(...params) {
+  if (params.length === 0) return undefined;
+
+  return isArray(...params)
+    ? Math.min(...returnOneArray(params))
+    : Math.min(...params);
+}
+
+function copy(copiedArray) {
+  return isArray(copiedArray) ? [...copiedArray] : { ...copiedArray };
 }
 
-export function copy() {
+function reverseMerge(...params) {
+  return params.reduceRight((acc, val) => acc.concat(val));
```

Comment:
I think you can simplify this using spread, no need to use the `concat` nor `reduceRight`. It's a nice way, though!